task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -77,3 +76,7 @@ def stop(self):
 
     def get_event_queue(self):
         return self.execution_framework.event_queue
+
+    @abc.abstractmethod
+    def handle_offer(self, task_configs, offer):
```

Comment:
handle_offer kinda implies that there are a bunch of things we could do with the offer. But, in reality,  we are just gonna create tasks to execute. Can you rename it to something like get_mesos_tasks?